components/admin/requests/processing/TasksCard.tsx

```diff
@@ -254,11 +253,34 @@ export default function ProcessingTasksCard({ applicationId }: ProcessingTasksCa
               disabled={!reviewRequestCompleted}
               color="black"
               onClick={handleGenerateInvoice}
+              isLoading={generateInvoiceLoading}
+              loadingText="Generate document"
+              fontWeight="normal"
+              fontSize="14px"
```

Comment:
not sure why but button was getting weird styling during loading but to explicitly specifying here seemed to do the trick